README.md

```diff
@@ -1,5 +1,131 @@
 # AutoML
 
-We're designing the AutoML platform that would make it easy for non-technical people to build custom AI solutions and AI-powered process automation.
+We're designing the AutoML platform that would make it easy for non-technical people to build custom AI solutions and
+AI-powered process automation.
 
-Easy-to-use ML platforms leverage the time/value/knowledge trade-off in a genuinely attractive way and allow users with no AI coding skills to optimize day-to-day operations and to solve business issues.
+Easy-to-use ML platforms leverage the time/value/knowledge trade-off in a genuinely attractive way and allow users with
+no AI coding skills to optimize day-to-day operations and to solve business issues.
+
+## API
+
+API Link: Todo
+
+## How Minikube deploys AutoML
+
+AutoML的部署脚本维护在``./deploy``目录下.
+按照以下指南在你本地Minikube上部署AutoML平台.
+
+### Prerequisites
+
+开始之前,请确保一下组件安装在你的机器上:
+
+- Docker
+- Minikube
+
+安装教程:
+
+**Mac OS:**
+
+- Docker: https://docs.docker.com/desktop/install/mac-install/
+
+- Minikube: ``brew install minikube``
+  - 详细参考:https://minikube.sigs.k8s.io/docs/start/
+
+**Linux:** Todo
+
+**Windows:** Todo
+
+#### Step 1: 部署依赖服务
+
+**MySQL部署**
+
+获取MySQL部署文件:
+
+- 获取mysql-pv.yaml：
+
+  ```bash
+  curl -O https://github.com/ml-for-good/automl/tree/main/deploy/mysql-pv.yaml
```

Comment:
Currently we can focus on deploy the folder 'deploy', we don't need to download from remote. Just need to clone the repo, then cd into the repo folder, then run the deploy command.